### ISSUE TITLE
The performance difference when target domain is  Clipart on office home dataset.

### ISSUE THREAD

**Issue by mzmzdcr** [uta-smile/TVT#9]
Hi！

Recently, when I was running your method, I encountered a problem. When I used baseline, the performance of all tasks whose target domain is Clipart decreased greatly. Can you help me solve this problem? It's more important to me.

![image](https://user-images.githubusercontent.com/38181266/180627132-f094cc89-c0d4-4442-978b-06378693f32f.png)

**Comment by viyjy** [comment]
Hi, thanks for you interest in our work.
Do you mean that you can get reasonable performance on other tasks? Clipart is the only exception?

**Comment by mzmzdcr** [comment]
Right, I encountered this strange phenomenon. Can you give me some advice？

**Comment by viyjy** [comment]
You only changed the batch size? Can you exactly follow the hyper-parameters as I used?

**Comment by mzmzdcr** [comment]
@viyjy. I reduced the bs to 32 and adjusted the input resolution to 224. Other hyperparameters are completely set by default

**Comment by viyjy** [comment]
why do you make these changes? Have you tried the command line in [script](https://github.com/uta-smile/TVT/blob/main/script.txt)?

**Comment by mzmzdcr** [comment]
I need to control the consumption of GPU memory below 24g, so I use this setting.

**Comment by viyjy** [comment]
can you also decrease the lr by half and see what happens?

**Comment by mzmzdcr** [comment]
Let me test whether it is feasible to reduce the learning rate

**Comment by mzmzdcr** [comment]
I'm sorry to reply to you so slowly because recently there was a problem with our GPU server device. First, whether the learning rate is increased or decreased, the results obtained on the task whose target domain is clipart are reduced.

Could you spare time to test locally and share the training log with me? The specific settings are as follows: 
--img_size 224 --train_batch_size 32 --beta 0.1 --gamma 0. --use_im --theta 0.1

**Comment by miscedence12** [comment]
@viyjy 请问当时你在office31数据集上训练一次要长时间呢？我将Epoch设置为200，显示需要12个小时，batch_size=64
